courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>I'd import the "Model" at the beginning to avoid "tf.keras". </p><p> </p><p>I'd package the model building into its own function that returns a compiled model: e.g.,</p><p> </p><p>def build_model(hidden_units):</p><p>  # etc.</p><p>  model = Model(inputs=etc.)</p><p>  model.compile(optimizer="adam", loss="mse", metrics=[rmse, "mse"])</p><p>  return model</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>